resources/tentative/complex-static-html/src/big-dom-generator/params.js

```diff
@@ -0,0 +1,12 @@
+// Default seed for the random number generator.
+export const SEED = 4212021;
+
+// Constrains for the generated DOM tree.
+export const MAX_DEPTH = 40;
+export const MAX_BREADTH = 16;
+export const CHILD_PROB = 0.7;
+export const TARGET_SIZE = 3000;
+
+// Maximum length of a generated CSS selector.
+export const MAX_SELECTOR_LEN = 7;
```

Comment:
Instead of adding a comment, give the constant more descriptive name:
e.g. MAX_SELECTOR_LENGTH_TO_GENERATE.